test/parse.js

```diff
@@ -389,6 +389,41 @@ var tests = [{
     expected: [
         { type: 'function', sourceIndex: 0, value: 'url', before: '', after: '', nodes: [] }
     ]
+}, {
+    message: 'should parse comments',
+    fixture: '/*before*/ 1px /*between*/ 1px /*after*/',
+    expected: [
+        { type: 'comment', sourceIndex: 0, value: 'before'},
+        { type: 'space', sourceIndex: 10, value: ' ' },
+        { type: 'word', sourceIndex: 11, value: '1px' },
+        { type: 'space', sourceIndex: 14, value: ' ' },
+        { type: 'comment', sourceIndex: 15, value: 'between'},
+        { type: 'space', sourceIndex: 26, value: ' ' },
+        { type: 'word', sourceIndex: 27, value: '1px' },
+        { type: 'space', sourceIndex: 30, value: ' ' },
+        { type: 'comment', sourceIndex: 31, value: 'after'},
+    ]
+}, {
+    message: 'should parse comments inside functions',
+    fixture: 'url( "/demo/bg.png" /*comment*/ )',
```

Comment:
It wasn't. But it's often case. Url can contain data:uri. That's why we should optimize a little this process.